.github/workflows/main.yml

```diff
@@ -24,3 +24,14 @@ jobs:
       - uses: ./go.mk/.github/actions/pre-check
 
       - run: make test-verbose
+
+      - name: Check for changes
+        id: check_changes
+        run: |
+          make generate
```

Comment:
`make generate` fetches latest openapi spec before running `go generate`.
Since this check runs on every commit, it will force us to commit changes that may not be relevant to the PR.
Let's instead execute:
```
cd v3/generator
go generate
```
Basically we just skip `wget <latest-oapi-spec>`.